### PR TITLE
[Merged by Bors] - chore(Algebra/Group/TypeTags): Remove porting notes related to lean4#1910, use new notation everywhere

### PR DESCRIPTION
**Pull request by edegeltje**
part 1: make applied occurances of `Additive.toAdd` and `Multiplicative.toMul` use newly available dot notation.

---

- [x] depends on: #19379

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Order/SuccPred/TypeTags.lean
Comment (edegeltje) [inline Mathlib/Algebra/Order/SuccPred/TypeTags.lean:1]: this had defeq abuse, so i change the type to be right.

Comment (eric-wieser) [reply Mathlib/Algebra/Order/SuccPred/TypeTags.lean:1]: Can you split that into a separate PR so that it's not hidden amongst all the dot notation stuff?

Comment (edegeltje) [reply Mathlib/Algebra/Order/SuccPred/TypeTags.lean:1]: done, see #19379

**Review by eric-wieser — APPROVED** [review]
bors d+

Please merge when CI passes

Thanks!

**Comment by eric-wieser** [comment]
Well, it passes

bors merge

**Pull request closed without merging**